app/views/email/CaptureEmailView.scala.html

```diff
@@ -14,69 +14,90 @@
-  }
-  <span id="form-hint" class="form-hint">@messages("captureEmail.hint")</span>
-}
-
-@mainTemplate(if(emailForm.errors.nonEmpty) messages("common.error.prefixTitle", messages("captureEmail.title")) else messages("captureEmail.title")) {
+@mainTemplate( pageTitle = if(emailForm.errors.nonEmpty) messages("common.error.prefixTitle", messages("captureEmail.title"))
+else messages("captureEmail.title")){
 
-    @if(letterToConfirmedEmail){
-        @currentEmail match {
-        case "" => {
-          <a class="link-back" href="@controllers.contactPreference.routes.AddEmailAddressController.show().url">@messages("base.back")</a>}
-        case _ => {
-          <a class="link-back" href="@controllers.contactPreference.routes.EmailToUseController.show().url">@messages("base.back")</a>}
-        }
+@if(letterToConfirmedEmail){
+    @currentEmail match {
+        case "" => { @govukBackLink(BackLink(
+        href = controllers.contactPreference.routes.AddEmailAddressController.show().url,
+        content = Text(messages("base.back"))
+    )) }
+case _ => {
+    @govukBackLink(BackLink(
+        href = controllers.contactPreference.routes.EmailToUseController.show().url,
+        content = Text(messages("base.back"))
+    ))
+    }
+}
+} else {@govukBackLink(BackLink(
+href = appConfig.dynamicJourneyEntryUrl(user.isAgent),
+content = Text(messages("base.back"))
+))
+}
 
-    } else {<a class="link-back" href="@appConfig.dynamicJourneyEntryUrl(user.isAgent)">@messages("base.back")</a>}
+@if(emailForm.hasErrors) {
+    @govukErrorSummary(ErrorSummary(
+        errorList = emailForm.errors.map { error =>
+        ErrorLink(content = Text(messages(error.message)), href = Some(s"#email"))
+    },
+    title = Text(messages("common.error.heading"))
+    ))
+}
 
+@formWithCSRF(action = submitFormAction) {
 
-        @errorSummary(messages("common.error.heading"), emailForm)
+@govukInput(Input(
+    id = "email",
+    name = "email",
+    classes = "govuk-!-width-one-half",
+    label = Label(
+        isPageHeading = true,
+        classes = "govuk-label--l",
+        content = Text(messages("captureEmail.title"))
+    ),
+    value = emailForm("email").value,
+    hint = Some(Hint(content = Text(messages("captureEmail.hint")))),
+    errorMessage = emailForm("email").error.map { err =>
+        ErrorMessage(content = Text(messages(err.message)))
+        }
+    ))
 
-        @form(action = submitFormAction) {
-            @text(
-                field = emailForm("email"),
-                pageTitle = Some(messages("captureEmail.title")),
-                additionalContent = Some(additionalContent)
-            )
+@if(currentEmail == "") {
+    <p class="govuk-body">@messages("captureEmail.onlyAddEmail")</p>
+}
 
-            <div class="form-group">
-                <button class="button" type="submit">@messages("common.continue")</button>
-            </div>
-        }
+@govukButton(Button(
+    content = Text(messages("common.continue")),
```

Comment:
Could you remove the comma from the end of this line as there's no param after it